spec/features/api_spec.rb

```diff
@@ -89,8 +89,29 @@
           }
           expect(json_response).to eq(expected)
         end
+
       end
 
+      describe "exporting csv" do
+	  		before do
```

Comment:
hmm.  Maybe Atom only fixes tabs to spaces when you type, not on paste? might have been something odd introduced by the git merge/rebase